Prototyping/HappyKiosk/Happy-Kiosk_Spawner.js

```diff
@@ -0,0 +1,588 @@
+// Happy-Kiosk_Spawner.js
+//
+// Created by Milad Nazeri on 2018-06-19
+//
+// Copyright 2018 High Fidelity, Inc.
+//
+// Distributed under the Apache License, Version 2.0.
+// See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+// Currently custom made for Rust content set (relies on Phlash's tables for layout)
+
+(function () {
+    // Helper Functions
+    var Util = Script.require("../../Utilities/Helper.js?");
+
+    var cacheBuster = Util.Scripts.cacheBuster,
+        findSurfaceBelowPosition = Util.Maths.findSurfaceBelowPosition,
+        getNameProps = Util.Entity.getNameProps,
+        getUserData = Util.Entity.getUserData,
+        inFrontOf = Util.Avatar.inFrontOf,
+        makeColor = Util.Color.makeColor,
+        updateUserData = Util.Entity.updateUserData,
+        vec = Util.Maths.vec;
+
+    // Log Setup
+    var LOG_ENTER = Util.Debug.LOG_ENTER,
+        LOG_UPDATE = Util.Debug.LOG_UPDATE,
+        LOG_ERROR = Util.Debug.LOG_ERROR,
+        LOG_VALUE = Util.Debug.LOG_VALUE,
+        LOG_ARCHIVE = Util.Debug.LOG_ARCHIVE, 
+        LOG_CONFIG = {
+            "Log_Enter": true,
+            "Log_Update": true,
+            "Log_Error": true,
+            "Log_Value": true,
+            "LOG_ARCHIVE": false
+        },
+        log = Util.Debug.log(LOG_CONFIG);  
+    
+    // Init
+    var BASE_NAME = "HappyKiosk_",
+        baseURL = "https://hifi-content.s3.amazonaws.com/milad/ROLC/Organize/O_Projects/Hifi/Scripts/hifi-content/Prototyping/HappyKiosk/",
```

Comment:
Can this be relative?

Reply:
I always use relative for all contained scripts except for spawner scripts to help testing staging/production versions.  

If it's more of a security concern with publishing the full path, I can definitely adjust this workflow to always use relative paths.